### ISSUE TITLE
CW for personal posts

### ISSUE THREAD

**Issue by alxsimon** [alxsimon/ecoevo.social-docs#18]
Suggested on Mastodon and also something I thought about adding.

CW for more personal/family posts.

It depends a lot on how much "science only" we want impose it to be.
Do people come to Mastodon (or Twitter before) just to find science? I'm not sure.

I'm half a mind for, and the other against.
Maybe do a poll on Mastodon at some point.

**Comment by naupaka** [comment]
I think this goes both ways. Some people only want science, but I also think it's important that people are seen as whole people and not just science. If there was a norm for some kind of hashtag or something, then those who didn't want to see those posts could turn on a filter for themselves.

**Comment by alxsimon** [comment]
Yes also my point of view. The hashtag filtering is a good idea.

**Comment by ethanwhite** [comment]
This feels problematic to me if we want to create a diverse and inclusive community. It feels counter to the idea of "bring your whole self" and I worry it would end up being a rule that institutionalizes the "just stick to the science" critiques that are often directed at minoritized groups speaking out about broader issues in the sciences and their everyday lives from their science focused social media accounts. Mastodon already has challenges in this space (see, e.g., [this Twitter thread by Chanda Prescod-Weinstein](https://twitter.com/IBJIYONGI/status/1590462702983720962) and this [nice post by a Mastodon vet](https://scholar.social/@CaribenxMarciaX/109313395721835492)). I'd like to avoid us reinforcing them.

**Comment by alxsimon** [comment]
Thanks for looking into this.
Also worth reading scholar.social about page https://scholar.social/about/more#regarding-attempts-to-abuse-these-anti-abuse-tools-and-policies. And CW on same page.

**Comment by castedo** [comment]
> Do people come to Mastodon (or Twitter before) just to find science?

Not sure how many, but I know some exist because I'm one of them! :smile: I also know there are people interested in science who avoid or barely use Twitter because it is full of too much non-science stuff. A related question is "How many people will start to follow evoeco.social scientists on Mastodon that were not following them (or were muting them) on Twitter?"

I'm only one data point, but I can honestly say that any tool that helps me read only what scientists have to say about science, and not their whole selves (roughly speaking), I would use. Today those tools are mainly:
1) the Twitter algos by clicking "Not interested in this tweet", "Stop recommending blahs by so-and-so", etc...
2) just not follow scientists that I can see from their history post about politics and personal stuff, etc...
3) silently muting folks because I feel bad unfollowing them because we know each other :sweat_smile:
4) Not actually follow the people I "follow" in Mastodon, but defacto follow the people I add/remove privately from a List I curate based on how informative, science focused, non-personal, and not-"whole self" the user is posting

I'm not saying anybody should mandate whether or not people bring their whole selves or post personal stuff or post political stuff, keep two accounts, CW or not, etc... Diff people will pick what they think is best for their situation. Totally reasonable for folks to think Mastodon and Twitter should be used for _SOCIAL_ reasons, where people are their whole selves, and I'm being an anti-social meanie. I'm just being honest sharing my own behavior FWIW. I might be a rare wierdo in that I'm not an academic nor employed by any lab. Twitter/Mastodon has been one of the only ways I can hear about research topics and news that academics would normally hear from co-workers, colleagues, lab mates, PIs, students, etc...

... I'm also finding Mastodon an _excellent_ way to procrastinate. :sweat_smile:
